.circleci/config.yml

```diff
@@ -133,12 +133,16 @@ references:
   upload_build_artifacts: &upload_build_artifacts
     run:
       name: Upload build artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+        aws s3api put-object-acl --acl public-read --bucket ${S3_BUILDS_BUCKET:?} --key ${PACKAGE_TARBALL_FILE:?}
```

Comment:
I prefer `basename ${PACKAGE_TARBALL:?}` instead of additional redundant variable that also pollutes the environment variables.